ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md

```diff
@@ -77,19 +77,19 @@ Für die Ressource Slot MUSS die REST-Interaktion "READ" implementiert werden.
     Anwendungshinweise: In diesem Fall ist auch ein Chaining auf weitere verknüpfte Akteure möglich: `GET https://example.org/fhir/Slot?schedule.actor:HealthcareService.type=https://example.org/fhir/CodeSystem/Behandlungsleistung|CT`
     Weitere Informationen zur Suche nach "Slot.schedule" finden sich in der [FHIR-Basisspezifikation - Abschnitt "reference"](https://hl7.org/fhir/R4/search.html#reference).
 
-3. Der Suchparameter "status" MUSS in Kombination mit einer Chaining-Abfrage auf Schedule unterstützt werden:
+3. Der Suchparameter "status" MUSS in Kombination ('&') mit einer Parameter-Abfrage auf Schedule unterstützt werden:
```

Comment:
```suggestion
3. Der Suchparameter "status" MUSS in Kombination ('&') mit einer Abfrage auf Schedule unterstützt werden. Diese Abfrage kann entweder eine direkt Angabe einer Referenz oder eine Angabe von weiteren Chaining-Parametern sein:
```

Reply:
Größtmögliche Freiheit in der Umsetzung erscheint mir sinnvoll. Was sagst Du, @alexey-tschudnowsky ?

Reply:
Den Vorschlag von @alexzautke finde ich gut